src/content/preview/Separation_of_concerns_in_react_apps.md

```diff
@@ -0,0 +1,344 @@
+---
+title: 'Separation of concerns in modern React apps - key principles and examples'
+date: 2021-02-12
+description: 'Building React apps, ever wondered where to put that "thing", and how to name it?'
+author: 'Rafał Świętek'
+authorAvatar: ./assets/avatar_rafal_swietek.png
+metaImage: ./assets/Separation_of_concerns_in_react_apps/meta_image.png
+metaTitle: 'Separation of concerns in modern React apps - key principles and examples'
+metaDescription: 'Building React apps, ever wondered where to put that "thing", and how to name it?'
+isFeature: false
```

Comment:
**T:** make it `true` if you want to be featured at the grey background. Remember to set `false` to the previous one.

Reply:
I'll update that when we'll be publishing it, thanks :)